WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -4313,18 +4406,49 @@ e_germania = {
 				romano_persian = "Iuliomontium"
 				romano_frank = "Iuliomontium"
 				romano_gallic = "Iuliomontium"
+				african_romance = "Iuliomontium"
+				dalmatian = "Iuliomontium"
+				pannonian = "Iuliomontium"
+				noricroman = "Iuliomontium"
+				raetoroman = "Iuliomontium"
+				latin = "Iuliomontium"
 				old_slavic = "Predhradi"
 				alamannian = "Julberg"
 				lombard = "Julberg"
 				rygir = "Fa�rguniba�rgs"
 				heruli = "Fa�rguniba�rgs"
 				ostrogothic = "Fa�rguniba�rgs"
 				romano_gothic = "Fa�rguniba�rgs"
-				polish = O�omuniec
+				polish = Olomuniec
```

Comment:
It should be "Ołomuniec".

Reply:
How do you add that sign?

Reply:
What do you mean?

Reply:
We said that if we could not add that symbol it was better to have just the regular L rather than the oxygen sign currently in the mod.